### PR TITLE
Add SEO & fix author tag

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
@svareg @rbovard I added the keywords and description for all pages, If you want more keywords or change descriptions let me know ! 
Maybe the german must be corrected !

File: content/pages/1_functionalities.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-Authors: Michael Kalbermatten, Rémi Bovard
+Author: Michael Kalbermatten, Rémi Bovard
 URL: functionalities
 Save_as: functionalities.html
 Slug: 1_functionalities
 Lang: en
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish is an application with a responsive ergonomics and a wide choice of functionalites
+
```
Comment (rbovard) [inline content/pages/1_functionalities.md]: Copy paste issue?

File: content/pages/0_index_fr.md [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 Menulabel: Accueil
+Tags: Geoportal, Open Source, GeoMapFish, SIG, Géospatial
+Summary: Geomapfish est une plateforme Open Source pour le développement de système d'information géographique en ligne (WebSIG) qui fournit un géoportail
 <br />
 Bienvenue sur le site web de la communauté GeoMapFish (GMF), merci de votre visite !
 
 ## Généralités
 
-GeoMapFish permet de construire des applications WebSIG de manière aisée et flexible.
+GeoMapFish permet de construire des applications WebSIG de manière aisée et flexible à travers un géoportai en ligne.
```
Comment (rbovard) [inline content/pages/0_index_fr.md]:
```suggestion
GeoMapFish permet de construire des applications WebSIG de manière aisée et flexible à travers un géoportail en ligne.
```

File: content/pages/1_functionalities_fr.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
 Date: 2016-10-14
 Modified: 2020-03-19
-Authors: Michael Kalbermatten, Rémi Bovard
+Author: Michael Kalbermatten, Rémi Bovard
 URL: functionalities
 Save_as: functionalities.html
 Slug: 1_functionalities
 Lang: fr
+Tags: Geoportal, Open Source, GeoMapFish, SIG, Géospatial
+Summary: Geomapfish est une application avec une érgonomie multi-supports et avec un large choix de fonctionnalités
```
Comment (rbovard) [inline content/pages/1_functionalities_fr.md]:
```suggestion
Summary: GeoMapFish est une application avec une ergonomie multi-supports et avec un large choix de fonctionnalités 
```

File: content/pages/2_documentation_fr.md [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
 Date: 2021-01-12
 Modified: 2021-01-12
-Authors: Rémi Bovard
+Author: Rémi Bovard
 URL: documentation
 Save_as: documentation.html
 Slug: 2_documentation
 Lang: fr
+Tags: Geoportal, Open Source, GeoMapFish, SIG, Géospatial
+Summary: Geomapfish dispose d'une large documentation technique pour les adminisatrateurs, intégrateurs et développeurs
```
Comment (rbovard) [inline content/pages/2_documentation_fr.md]:
```suggestion
Summary: GeoMapFish dispose d'une large documentation technique pour les adminisatrateurs, intégrateurs et développeurs
```

File: content/pages/3_gallery.md [hunk 5]
```diff
@@ -1,11 +1,13 @@
...
 Date: 2016-10-14
 Modified: 2021-01-19
-Authors: Michael Kalbermatten, Rémi Bovard, Olivier Monod
+Author: Michael Kalbermatten, Rémi Bovard, Olivier Monod
 URL: gallery
 Save_as: gallery.html
 Slug: 3_gallery
 Lang: en
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish is already functional on many geoportal
```
Comment (rbovard) [inline content/pages/3_gallery.md]:
```suggestion
Summary: GeoMapFish is already functional on many geoportal
```

File: content/pages/0_index.md [hunk 6]
```diff
@@ -1,18 +1,20 @@
...
 Modified: 2018-05-16
-Authors: Michael Kalbermatten, Rémi Bovard
+Author: Michael Kalbermatten, Rémi Bovard
 URL: index.html
 Save_as: index.html
 Slug: 0_index
 Lang: en
 Menulabel: Home
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish is an Open Source platform for the development of online geographic information systems (WebGIS) that provide a geoportal
```
Comment (rbovard) [inline content/pages/0_index.md]:
```suggestion
Summary: GeoMapFish is an Open Source platform for the development of online geographic information systems (WebGIS) that provide a geoportal
```

File: content/pages/0_index_de.md [hunk 7]
```diff
@@ -1,16 +1,18 @@
...
 Modified: 2018-05-16
-Authors: Michael Kalbermatten, Irene Vontobel, Rémi Bovard
+Author: Michael Kalbermatten, Irene Vontobel, Rémi Bovard
 URL: index.html
 Save_as: index.html
 Slug: 0_index
 Lang: de
 Menulabel: Startseite
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish ist eine Open-Source Plattform für die Entwicklung eines Online Geoinformationssystems (WebGIS), das ein Geoportal bereitstellt
```
Comment (rbovard) [inline content/pages/0_index_de.md]:
```suggestion
Summary: GeoMapFish ist eine Open-Source Plattform für die Entwicklung eines Online Geoinformationssystems (WebGIS), das ein Geoportal bereitstellt
```

File: content/pages/0_index_fr.md [hunk 8]
```diff
@@ -1,18 +1,20 @@
...
 Modified: 2018-05-16
-Authors: Michael Kalbermatten, Rémi Bovard
+Author: Michael Kalbermatten, Rémi Bovard
 URL: index.html
 Save_as: index.html
 Slug: 0_index
 Lang: fr
 Menulabel: Accueil
+Tags: Geoportal, Open Source, GeoMapFish, SIG, Géospatial
+Summary: Geomapfish est une plateforme Open Source pour le développement de système d'information géographique en ligne (WebSIG) qui fournit un géoportail
```
Comment (rbovard) [inline content/pages/0_index_fr.md]:
```suggestion
Summary: GeoMapFish est une plateforme Open Source pour le développement de système d'information géographique en ligne (WebSIG) qui fournit un géoportail
```

File: content/pages/1_functionalities.md (hunk 1 above)
Comment (rbovard) [reply content/pages/1_functionalities.md]: `GeoMapFish`

File: content/pages/1_functionalities_de.md [hunk 9]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2016-10-14
 Modified: 2020-03-19
-Authors: Michael Kalbermatten, Irene Vontobel, Rémi Bovard
+Author: Michael Kalbermatten, Irene Vontobel, Rémi Bovard
 URL: functionalities
 Save_as: functionalities.html
 Slug: 1_functionalities
 Lang: de
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish ist eine Anwendung mit Ergonomie für jedes digitale Medium und einer Vielzahl von Funktionen
```
Comment (rbovard) [inline content/pages/1_functionalities_de.md]:
```suggestion
Summary: GeoMapFish ist eine Anwendung mit Ergonomie für jedes digitale Medium und einer Vielzahl von Funktionen
```

File: content/pages/2_documentation.md [hunk 10]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2021-01-12
 Modified: 2021-01-12
-Authors: Rémi Bovard
+Author: Rémi Bovard
 URL: documentation
 Save_as: documentation.html
 Slug: 2_documentation
 Lang: en
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish has wide technical documentation for administrators, integrators and developers
```
Comment (rbovard) [inline content/pages/2_documentation.md]:
```suggestion
Summary: GeoMapFish has wide technical documentation for administrators, integrators and developers
```

File: content/pages/2_documentation_de.md [hunk 11]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2021-01-12
 Modified: 2021-01-12
-Authors: Rémi Bovard
+Author: Rémi Bovard
 URL: documentation
 Save_as: documentation.html
 Slug: 2_documentation
 Lang: de
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish verfügt über eine umfangreiche Dokumentationstechnik für Administratoren, Integratoren und Entwickler
```
Comment (rbovard) [inline content/pages/2_documentation_de.md]:
```suggestion
Summary: GeoMapFish verfügt über eine umfangreiche Dokumentationstechnik für Administratoren, Integratoren und Entwickler
```

File: content/pages/3_gallery_de.md [hunk 12]
```diff
@@ -1,11 +1,13 @@
...
 Date: 2016-10-14
 Modified: 2021-01-19
-Authors: Michael Kalbermatten, Rémi Bovard, Olivier Monod
+Author: Michael Kalbermatten, Rémi Bovard, Olivier Monod
 URL: gallery
 Save_as: gallery.html
 Slug: 3_gallery
 Lang: de
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish ist bereits auf vielen Geoportalen funktionsfähig
```
Comment (rbovard) [inline content/pages/3_gallery_de.md]:
```suggestion
Summary: GeoMapFish ist bereits auf vielen Geoportalen funktionsfähig
```

File: content/pages/3_gallery_fr.md [hunk 13]
```diff
@@ -1,11 +1,13 @@
...
 Date: 2016-10-14
 Modified: 2021-01-19
-Authors: Michael Kalbermatten, Rémi Bovard, Olivier Monod
+Author: Michael Kalbermatten, Rémi Bovard, Olivier Monod
 URL: gallery
 Save_as: gallery.html
 Slug: 3_gallery
 Lang: fr
+Tags: Geoportal, Open Source, GeoMapFish, SIG, Géospatial
+Summary: Geomapfish est déjà fonctionel sur de nombreux géoportails
```
Comment (rbovard) [inline content/pages/3_gallery_fr.md]:
```suggestion
Summary: GeoMapFish est déjà fonctionel sur de nombreux géoportails
```

File: content/pages/4_community.md [hunk 14]
```diff
@@ -1,11 +1,13 @@
...
 Date: 2018-05-16
 Modified: 2019-10-02
-Authors: Michael Kalbermatten, Rémi Bovard, Olivier Monod
+Author: Michael Kalbermatten, Rémi Bovard, Olivier Monod
 URL: community
 Save_as: community.html
 Slug: 4_community
 Lang: en
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish is part of a community with members coming from the public, private and academic sectors
```
Comment (rbovard) [inline content/pages/4_community.md]:
```suggestion
Summary: GeoMapFish is part of a community with members coming from the public, private and academic sectors
```

File: content/pages/4_community_de.md [hunk 15]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2018-05-16
 Modified: 2019-10-02
-Authors: Michael Kalbermatten, Rémi Bovard, Olivier Monod
+Author: Michael Kalbermatten, Rémi Bovard, Olivier Monod
 URL: community
 Save_as: community.html
 Slug: 4_community
 Lang: de
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish ist Teil einer Community mit Mitgliedern aus dem öffentlichen, privaten und akademischen Bereich
```
Comment (rbovard) [inline content/pages/4_community_de.md]:
```suggestion
Summary: GeoMapFish ist Teil einer Community mit Mitgliedern aus dem öffentlichen, privaten und akademischen Bereich
```

File: content/pages/4_community_fr.md [hunk 16]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2018-05-16
 Modified: 2019-10-02
-Authors: Michael Kalbermatten, Rémi Bovard, Olivier Monod
+Author: Michael Kalbermatten, Rémi Bovard, Olivier Monod
 URL: community
 Save_as: community.html
 Slug: 4_community
 Lang: fr
+Tags: Geoportal, Open Source, GeoMapFish, SIG, Géospatial
+Summary: Geomapfish fait partie d'une communauté composée de membres issus du secteur public, privé et académique
```
Comment (rbovard) [inline content/pages/4_community_fr.md]:
```suggestion
Summary: GeoMapFish fait partie d'une communauté composée de membres issus du secteur public, privé et académique
```

File: content/pages/5_roadmap.md [hunk 17]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2017-06-20
 Modified: 2020-09-23
-Authors: Michael Kalbermatten, Rémi Bovard
+Author: Michael Kalbermatten, Rémi Bovard
 URL: roadmap
 Save_as: roadmap.html
 Slug: 5_roadmap
 Lang: en
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish is an Open Source platform that evolves with each version by offering many new functionalities
```
Comment (rbovard) [inline content/pages/5_roadmap.md]:
```suggestion
Summary: GeoMapFish is an Open Source platform that evolves with each version by offering many new functionalities
```

File: content/pages/5_roadmap_de.md [hunk 18]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2017-06-20
 Modified: 2020-09-23
-Authors: Michael Kalbermatten, Rémi Bovard, Irene Vontobel
+Author: Michael Kalbermatten, Rémi Bovard, Irene Vontobel
 URL: roadmap
 Save_as: roadmap.html
 Slug: 5_roadmap
 Lang: de
+Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
+Summary: Geomapfish ist eine Open-Source Plattform, die sich mit jeder Version weiterentwickelt, indem sie viele neue Funktionalitäten bietet
```
Comment (rbovard) [inline content/pages/5_roadmap_de.md]:
```suggestion
Summary: GeoMapFish ist eine Open-Source Plattform, die sich mit jeder Version weiterentwickelt, indem sie viele neue Funktionalitäten bietet
```

File: content/pages/5_roadmap_fr.md [hunk 19]
```diff
@@ -1,11 +1,14 @@
...
 Date: 2017-06-20
 Modified: 2020-09-23
-Authors: Michael Kalbermatten, Rémi Bovard
+Author: Michael Kalbermatten, Rémi Bovard
 URL: roadmap
 Save_as: roadmap.html
 Slug: 5_roadmap
 Lang: fr
+Tags: Geoportal, Open Source, GeoMapFish, SIG, Géospatial
+Summary: Geomapfish est une plateforme Open Source qui évolue à chaque version en proposant un lot de nouvelles fonctionnalités
```
Comment (rbovard) [inline content/pages/5_roadmap_fr.md]:
```suggestion
Summary: GeoMapFish est une plateforme Open Source qui évolue à chaque version en proposant un lot de nouvelles fonctionnalités 
```

**Review by rbovard — CHANGES_REQUESTED** [review]
Thanks for the work!

Is there a way to see the result somewhere?

I noticed some typos when reviewing FR and EN versions.

@svareg Can you please check DE version?

File: content/pages/1_functionalities.md (hunk 1 above)
Comment (pfanguin) [reply content/pages/1_functionalities.md]: Yes I think !

Comment (pfanguin) [inline content/pages/1_functionalities.md]:
```suggestion
Tags: Geoportal, Open Source, GeoMapFish, GIS, Geospatial
Summary: GeoMapFish is an application with a responsive ergonomics and a wide choice of functionalites
```

**Comment by pfanguin** [comment]
@rbovard no I don't have a link for testing, sorry... but it's work fine (see attached image)
image

**Review by rbovard — APPROVED** [review]

**Comment by remyguillaume** [comment]
Hello,

There is no problem for me with the german version... for my german level in any case :-)
Thank to you both !

**Pull request merged**